app/bundles/LeadBundle/Form/Type/CampaignEventLeadStagesType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
```

Comment:
Declare strict types for all new fields, please.
```suggestion

declare(strict_types=1);

```